src/main/java/servlets/module/challenge/SessionManagement4.java

```diff
@@ -98,7 +98,7 @@ public void doPost (HttpServletRequest request, HttpServletResponse response)
 					{
 						log.debug("Guest Session Detected");
 					}
-					else if (decodedCookie.equals("0000000000000021")) //Admin Session
+					else if (decodedCookie.equals("0000000000000009")) //Admin Session

```

Comment:
Any reason why this was changed from 0000000000000021 to 0000000000000009? Makes the level a bit easier?
If this change sticks it's rank/points should be reassessed

Reply:
Please see #529. Agreed on changing rank/points if needed.